dom.bs

```diff
@@ -6373,17 +6376,32 @@ given a <var>document</var>, <var>localName</var>, <var>namespace</var>, and opt
 <ol>
  <li><p>Let <var>oldValue</var> be <var>attribute</var>'s <a for=Attr>value</a>.</p></li>
 
- <li><p>Set <var>attribute</var>'s <a for=Attr>value</a> to <var>value</var>.
+ <li><p><a>Verify and set attribute value</a> <var>value</var> for <var>attribute</var>, with
+ <var>attribute</var>'s <a for=Attr>element</a>.
 
- <li><p><a>Handle attribute changes</a> for <var>attribute</var> with <var>attribute</var>'s
- <a for=Attr>element</a>, <var>oldValue</var>, and <var>value</var>.
+ <li><p>If <var>attribute</var>'s <a for=Attr>element</a> <a lt="has an attribute">has</a>
+ an <a>attribute</a> <var>attribute</var>, then <a>handle attribute changes</a> for
```

Comment:
"has an attribute" should be the linking text I think. Not sure why we'd mention the type as its own thing here.